src/vivarium_gates_bep/components/treatment.py

```diff
@@ -123,9 +126,9 @@ def setup(self, builder):
 
     def adjust_lbwsg(self, index, exposure):
         pop = self.population_view.get(index)
-        baseline_covered = pop[project_globals.BASELINE_COLUMN] == project_globals.TREATMENTS.IFA
-        exposure.loc[baseline_covered, project_globals.BIRTH_WEIGHT] -= self.treatment_effects[project_globals.TREATMENTS.IFA]
 
+        p_ifa = load_ifa_proportion_among_anc(self.draw, self.location)
```

Comment:
Load this once during setup.  No need to repeatedly recompute.

Reply:
Done.